x-pack/plugins/triggers_actions_ui/public/application/lib/action_connector_api.ts

```diff
@@ -11,7 +11,7 @@ import type { ActionConnector, ActionConnectorWithoutId, ActionType } from '../.
 import { ActionTypeExecutorResult } from '../../../../../plugins/actions/common';
 
```

Comment:
These API calls will now have snake case responses. Should we handle the conversion back to camel case in this file?